src/data/AmazonReviewData.py

```diff
@@ -17,11 +17,12 @@
 class AmazonReviewFullDataModule(pl.LightningDataModule):
     def __init__(
         self,
-        tokenizer,
+        tokenizer=None,
         val_size: float = 0.2,
```

Comment:
Add val_size as a class attribute 

```suggestion
        val_size: float = 0.2,
        
        self.val_size = val_size
```